### ISSUE TITLE
Invalid Mandatory prop error introduced by 9eae86

### ISSUE THREAD

**Issue by gtmills** [DMTF/Redfish-Service-Validator#379]
OpenBMC CI uses the latest master of Redfish-Service-Validator.
Yesterday we started seeing: 

`1 failMandatoryExist errors in /redfish/v1/AccountService/Accounts/root`
`ERROR - AccountTypes: Mandatory prop does not exist`

This is not a valid error though since OpenBMC is using #ManagerAccount.v1_3_0.ManagerAccount which does not have the AccountTypes property. 

From https://redfish.dmtf.org/schemas/ManagerAccount.v1_6_0.json:
```
                "AccountTypes": {
                    "description": "The account types.",
                    "items": {
                        "anyOf": [
                            {
                                "$ref": "#/definitions/AccountTypes"
                            },
...
                    "versionAdded": "v1_4_0"
                },
```
Determined this was introduced by:
https://github.com/DMTF/Redfish-Service-Validator/commit/9eae8612fe81a73df93942edacd90de2bbea29a5

Have a workaround in OpenBMC (move to ManagerAccount.v1_4_0 and implement AccountTypes),
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/33542 but would like to see this fixed. 

Thanks!!!
Gunnar

**Comment by gtmills** [comment]
Logs: 
[Redfish-378.zip](https://github.com/DMTF/Redfish-Service-Validator/files/4771862/Redfish-378.zip)

**Comment by gtmills** [comment]
@mraineri @tomasg2012

**Comment by tomasg2012** [comment]
I can't seem to reproduce this so easily, would you be able to do a secondary run before the change on the service was implement, but with `--debug_logging` enabled

**Comment by gtmills** [comment]
With --debug_logging:
[redfish-debug-378.zip](https://github.com/DMTF/Redfish-Service-Validator/files/4772825/redfish-debug-378.zip)

If it helps:

```
 curl -k https://${bmc}/redfish/v1/AccountService/Accounts/root
{
  "@odata.id": "/redfish/v1/AccountService/Accounts/root",
  "@odata.type": "#ManagerAccount.v1_3_0.ManagerAccount",
  "Description": "User Account",
  "Enabled": true,
  "Id": "root",
  "Links": {
    "Role": {
      "@odata.id": "/redfish/v1/AccountService/Roles/Administrator"
    }
  },
  "Locked": false,
  "Locked@Redfish.AllowableValues": [
    "false"
  ],
  "Name": "User Account",
  "Password": null,
  "PasswordChangeRequired": false,
  "RoleId": "Administrator",
  "UserName": "root"
}
```

**Comment by tomasg2012** [comment]
Thank you!  I pushed a commit to `version-hotfix` and it should resolve this issue.

**Comment by gtmills** [comment]
Tested the version-hotfix branch and it appeared to fix the problem. thank you for the quick fix here!

**Comment by mraineri** [comment]
Fix merged to master (and released in 1.3.7)
